components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionManagementServiceImpl.java

```diff
@@ -38,6 +41,12 @@
 import java.util.Map;
 import java.util.UUID;
 
+import static org.wso2.carbon.identity.central.log.mgt.utils.LogConstants.ActionManagement.ACTIVATE_ACTION;
+import static org.wso2.carbon.identity.central.log.mgt.utils.LogConstants.ActionManagement.ADD_ACTION;
+import static org.wso2.carbon.identity.central.log.mgt.utils.LogConstants.ActionManagement.DEACTIVATE_ACTION;
+import static org.wso2.carbon.identity.central.log.mgt.utils.LogConstants.ActionManagement.DELETE_ACTION;
+import static org.wso2.carbon.identity.central.log.mgt.utils.LogConstants.ActionManagement.UPDATE_ACTION;
```

Comment:
IMO, we don't need to add these constants to the central log mgt unit. These are related to the component. So the component can define them

Reply:
Refactored with a105238b1c38f7405ecf43ecaa9c07b9b0de7b7f